features/create.feature

```diff
@@ -27,6 +27,17 @@ Feature: hub create
       """
     When I successfully run `hub create -p`
     Then the url for "origin" should be "git@github.com:mislav/dotfiles.git"
+  Scenario: Create repo with new remote name specified
+    Given the GitHub API server:
+    """
```

Comment:
Minor: the `"""` lines are usually indented one level under the `Given` clause (see other tests)